sources/packages/web/src/views/aest/student/FileUploads.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card class="mt-4">
+    <div class="mx-5 py-4">
+      <content-group>
+        <body-header
+          title="File Uploads"
+          :recordsCount="studentFileUploads?.length"
+          class="m-1"
+        >
+        </body-header>
+        <DataTable
+          :value="studentFileUploads"
+          :paginator="true"
+          :rows="DEFAULT_PAGE_LIMIT"
+          :rowsPerPageOptions="PAGINATION_LIST"
+        >
+          <template #empty>
+            <p class="text-center font-weight-bold">No records found.</p>
+          </template>
+          <Column
+            field="groupName"
+            header="Document Purpose"
+            sortable="true"
+          ></Column>
+          <Column field="metadata" header="Application #">
+            <template #body="slotProps"
+              ><span v-if="slotProps.data.metadata">{{
```

Comment:
IMO instead of using v-if and v-else with span we can use like this. 

[ {{slotProps.data.metadata.applicationNumber ? slotProps.data.metadata.applicationNumber : "-"} ]